src/System.Text.Encodings.Web/src/System/Text/Encodings/Web/HtmlEncoder.cs

```diff
@@ -24,7 +25,7 @@ public static HtmlEncoder Create(params UnicodeRange[] allowedRanges)
         }
     }
 
-    public sealed class DefaultHtmlEncoder : HtmlEncoder
+    internal sealed class DefaultHtmlEncoder : HtmlEncoder
```

Comment:
Yeah in this case I concede there are no interesting existing public entry points that are callable. I thought at one time we had discussed putting the default encoders in a sub-namespace `DefaultEncoders`. That namespace would presumably not normally be imported in a given file, and so a typical developer wouldn't have to "learn" it.